src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,46 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => {
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+  } = movie;
+
+  return (
+    <div className="card" data-cy="Movie">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img data-cy="MovieImage" src={imgUrl} alt="Film logo" />
```

Comment:
formatting. if you have more than 2 props/ attributes, put each of them on a new line